LaTeXTools.sublime-settings

```diff
@@ -246,7 +246,9 @@
 
 	"osx": 	{
 		// Path used when invoking tex & friends; MUST include $PATH
-		"texpath" : "/Library/TeX/texbin:/usr/texbin:/usr/local/bin:/opt/local/bin:$PATH"
+		"texpath": "/Library/TeX/texbin:/usr/texbin:/usr/local/bin:/opt/local/bin:$PATH",
+		// Command to invoke Sioyek. If blank, "sioyek.exe" is used (it has to be on your PATH)
+		"sioyek": ""
```

Comment:
```suggestion
		"sioyek": "",
```

So that next time one is added, you don't have to change the previous line, too.

Reply:
I just followed existing "style" of avoiding trailing commas. Maybe a general styling task.